samples/angular-app/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
+
```

Comment:
With the change to require explicit component registration you need to ensure you call the relevant register functions, this one wraps all of them, and isn't recommended for production use cases.

```suggestion
import { registerMgtComponents } from '@microsoft/mgt-components';

registerMgtComponents();
```